src/Jackett.Common/Indexers/DarmoweTorrenty.cs

```diff
@@ -0,0 +1,234 @@
+                var bannedUserElement = dom.QuerySelector("b:contains(\"has been blocked - fill captcha that\")");
+                var errorMessage = invalidLoginDiv?.TextContent ??
+                                   invalidPasswordDiv?.TextContent ??
+                                   bannedUserElement?.TextContent;
+                throw new ExceptionWithConfigData(errorMessage, configData);
+            });
+            return IndexerConfigurationStatus.RequiresTesting;
+        }
+
+        private ReleaseInfo ParseRow(IParentNode titleRow, IElement detailsRow)
+        {
+            var categoryAttribute = detailsRow.QuerySelector("a[href^=\"/torrenty.php?category=\"]").GetAttribute("href");
+            var categoryUrl = new Uri(SiteLink + Uri.UnescapeDataString(categoryAttribute));
+            var trackerCategory = HttpUtility.ParseQueryString(categoryUrl.Query)["category"];
+            var categories = MapTrackerCatToNewznab(trackerCategory);
+            var seedsMatch = _SeedsRegex.Match(detailsRow.TextContent);
+            var leechersMatch = _LeechersRegex.Match(detailsRow.TextContent);
+            var dateMatch = _DateRegex.Match(detailsRow.TextContent);
+            var sizeMatch = _SizeRegex.Match(detailsRow.TextContent);
+            var date = DateTime.MinValue; // In case of parsing failure
+            if (dateMatch.Success)
+            {
+                var polishDate = DateTime.SpecifyKind(DateTime.ParseExact(dateMatch.Groups[1].Value, "dd/MM/yyyy", CultureInfo.InvariantCulture), DateTimeKind.Unspecified);
+                var pubDateUtc = TimeZoneInfo.ConvertTimeToUtc(polishDate, _polandTz);
+                date = pubDateUtc.ToLocalTime();
+            }
+            var details = titleRow.QuerySelector("a[href^=\"details.php?id=\"]:has(span)");
+            var detailsLink = new Uri(SiteLink + details.GetAttribute("href"));
+            var encodedDownloadLink = detailsRow.QuerySelector("a[id^=\"download_\"]").GetAttribute("data-href");
+            var downloadLink = new Uri(SiteLink + Uri.UnescapeDataString(StringUtil.FromBase64(encodedDownloadLink)));
+            var bannerLink = detailsRow.QuerySelector("img[src^=\"./imgtorrent/\"]")?.GetAttribute("src");
+            var seeders = seedsMatch.Success ? int.Parse(seedsMatch.Groups[1].Value) : 0;
+            var leechers = leechersMatch.Success ? int.Parse(leechersMatch.Groups[1].Value) : 0;
+            var peers = seeders + leechers;
+            var release = new ReleaseInfo
+            {
+                Title = details.TextContent,
+                Category = categories,
+                Seeders = seeders,
+                BannerUrl = !string.IsNullOrEmpty(bannerLink) ? new Uri(SiteLink + bannerLink) : null,
+                Peers = peers,
+                PublishDate = date,
+                DownloadVolumeFactor = 0,
+                UploadVolumeFactor = 1,
+                Link = downloadLink,
```

Comment:
the download link is not working for those that have 0 points.
https://darmowe-torenty.pl/download.php?id=8301983d5f9e992c29f30df9c21eeb2a679b893e&f=sk%B3adanka+mireczka+++jazda+na+maxa+vol+7+%2A2020%2A+%5Bmp3+128+320+kbps%5D+%28m19%29.torrent
returns a html page instead of a torrent.

since the site is free, we should not have to pay for free stuff, so we build a magnet link instead, like the original yaml indexer did.

Reply:
```yaml
download:
  selector: table[align="center"][width="100%"] tbody tr:nth-child(2) td:nth-child(2)
  filters:
    - name: toupper
    - name: trim
    - name: prepend
      args: "magnet:?xt=urn:btih:"
    - name: append
      args: "&tr=udp://tracker.opentrackr.org:1337&tr=udp://tracker.coppersurfer.tk:6969&tr=udp://tracker.internetwarriors.net:1337&tr=udp://tracker.leechers-paradise.org:6969"
```

Reply:
fixed, a magnet link is being generated now.